endpoints/openrtb2/auction.go

```diff
@@ -824,7 +826,10 @@ func (deps *endpointDeps) validateRequest(account *config.Account, httpReq *http
 			return []error{err}
 		}
 
-		if err := deps.validateEidPermissions(reqPrebid.Data, requestAliases); err != nil {
+		err, warns := deps.validateEidPermissions(reqPrebid.Data, requestAliases, isDebugEnabled)
+		errL = append(errL, warns...)
```

Comment:
If this is a single response, it can always be appended to errL. The err != nil check below can use the `errortypes.ContainsFatalError` helper instead to check for fatal errors, aka non-warnings.